requirements.txt

```diff
@@ -0,0 +1,2 @@
+matplotlib
+torch>=1.8
```

Comment:
This is sufficient for AADL, but the examples have additional requirements as mentioned in the Issue.

We should add another `requirements.txt` for the examples, or maybe even one for each example.

Reply:
Good point. Is it good practice to add the additional `requirements.txt` in each directory, or should I add th additional requirement files at the same level as this one?

Reply:
I'm not sure what the common practice is, but I think putting things in the corresponding folder would be best.

We don't want users running just AADL to install all the example dependencies, and we don't want a pile of requirements in the main folder like `requirements-example1.txt`, `requirements-example2.txt` and so on.